recipes/festival/install.sh

```diff
@@ -0,0 +1,35 @@
+set -exou
+
+# All files have been built by build.sh, this just installs them into ${PREFIX}
+
+mkdir -p "${PREFIX}/bin"
+mkdir -p "${PREFIX}/lib"
+mkdir -p "${PREFIX}/lib/festival"
+mkdir -p "${PREFIX}/share/festival"
+
+# Prepare the dicts and voices directories. Dependent packages will put their dicts and voices there.
+mkdir -p "${PREFIX}/share/festival/dicts"
+mkdir -p "${PREFIX}/share/festival/voices"
+# Conda would delete the directories if they were left empty, so we use a placeholder file.
+echo "keep" > "${PREFIX}/share/festival/dicts/.keep"
+echo "keep" > "${PREFIX}/share/festival/voices/.keep"
```

Comment:
Does the package require these directories to exist even if they are empty?

Reply:
Yes, further in this script, I need to create the symlinks from share/festival to lib, and they are essential for the packages with voices. If these directories would be deleted, the symlinks would not work.